Xamarin.Forms.Build.Tasks/CompiledConverters/ListStringTypeConverter.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+
+using Xamarin.Forms.Xaml;
+using Xamarin.Forms.Build.Tasks;
+
+namespace Xamarin.Forms.Core.XamlC
+{
+	class ListStringTypeConverter : ICompiledTypeConverter
+	{
+		public IEnumerable<Instruction> ConvertFromString(string value, ModuleDefinition module, BaseNode node)
+		{
+			if (value == null) {
+				yield return Instruction.Create(OpCodes.Ldnull);
+				yield break;
+			}
+			var parts = value.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(s => s.Trim()).ToList();
```

Comment:
I wonder if there is a scenario where users would want a whitespace entry to occupy space in a list of strings. Perhaps providing labels for a chart where they want to skip every other item, for example?